battleship-logic/src/main/java/dreamteam/battleship/logic/movement/MovementManager.java

```diff
@@ -70,8 +69,8 @@ default MovementStatus tryPutShip(Ship ship, int fromFieldNumber, Direction dire
         return MovementStatus.INVALID_MOVEMENT;
     }
 
-    default int numberOfPlayerShots() throws Exception {
-        throw new InvalidActivityException("number of player can be called only in DamageManager");
+    default int numberOfPlayerShots() {
+        return 0;
```

Comment:
Why this method always return :1234:  `0`? Would it be changed in the future? You could write some `//TODO` and comment.